src/App.jsx

```diff
@@ -29,6 +30,16 @@ function KanbanBoard() {
   const [selectedTask, setSelectedTask] = useState(null);
   const emojiPickerRef = useRef(null);
 
+  const handleTaskClick = (columnId, task) => {
+    setSelectedTask({ ...task, columnId });
+  };
+
+  const handleTaskClickMemoized = useCallback((columnId, task, isDragging) => {
+    if (!isDragging) {
+      handleTaskClick(columnId, task);
+    }
+  }, []);
```

Comment:
![kody code-review](https://img.shields.io/badge/kody-code--review-brightgreen)

```suggestion
const handleTaskClickMemoized = useCallback((columnId, task, isDragging) => {
  if (!isDragging) {
    setSelectedTask({ ...task, columnId });
  }
}, []);
```
The handleTaskClickMemoized dependency array is empty but uses handleTaskClick which depends on setSelectedTask. Consider adding it to the dependency array or moving handleTaskClick inside useCallback to properly handle all dependencies.



<sub>Talk to Kody by mentioning @kody</sub>

<sub>Was this suggestion helpful? React with 👍 or 👎 to help Kody learn from this interaction.</sub><!-- kody-codereview -->&#8203;
&#8203;